snmp.yml

```diff
@@ -1118,6 +1141,66 @@ apcups:
       type: gauge
     - labelname: upsHighPrecBatteryCartridgeIndex
       type: gauge
+  - name: upsHighPrecBatteryPackOnlyTableSize
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.3
+    type: gauge
+    help: The number of entries in the upsHighPrecBatteryPacks. - 1.3.6.1.4.1.318.1.1.1.2.3.10.3
+  - name: upsHighPrecBatteryPackOnlyIndex
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.1
+    type: gauge
+    help: The battery pack identifier. - 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.1
+    indexes:
+    - labelname: upsHighPrecBatteryPackOnlyIndex
+      type: gauge
+  - name: upsHighPrecBatteryPackOnlyFirmwareRevision
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.2
+    type: DisplayString
+    help: The battery pack firmware revision. - 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.2
+    indexes:
+    - labelname: upsHighPrecBatteryPackOnlyIndex
+      type: gauge
+  - name: upsHighPrecBatteryPackOnlySerialNumber
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.3
+    type: DisplayString
+    help: The battery pack serial number. - 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.3
+    indexes:
+    - labelname: upsHighPrecBatteryPackOnlyIndex
+      type: gauge
+  - name: upsHighPrecBatteryPackOnlyTemperature
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.4
+    type: gauge
+    help: The battery pack temperature measured in 10ths of degree Celcius - 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.4
+    indexes:
+    - labelname: upsHighPrecBatteryPackOnlyIndex
+      type: gauge
+  - name: upsHighPrecBatteryPackOnlyStatus
+    oid: 1.3.6.1.4.1.318.1.1.1.2.3.10.4.1.5
+    type: OctetString
```

Comment:
Wondering whether this should be overridden to `DisplayString`

Reply:
Is it always an ASCII/UTF-8 string?

Reply:
Mmm, according to the MIB docs:

```
The battery status for the pack only.
bit 0 Disconnected
bit 1 Overvoltage
bit 2 NeedsReplacement
bit 3 OvertemperatureCritical
bit 4 Charger
bit 5 TemperatureSensor
bit 6 BusSoftStart
bit 7 OvertemperatureWarning
```

Reply:
That seems like it returns a single byte and you have to look at the bits yourself.

Reply:
Yeah. I guess so. So we leave it as such?

Reply:
Yea, I would leave it, it doesn't seem like a string, and without a device to test with it's hard to say.

Reply:
You might be able to do something with regex extracts.